README.md

```diff
@@ -1,260 +1,289 @@
+
+This readme explains how to set up and use this software. 
+In doing so, it uses data from the 2016 United States presidential election. 
+These data are included in the repository.
+
+**Note:** Any images generated using this software should be referenced to:
+
+Gastner, M., Seguy, V., & More, P. (2018). Fast flow-based algorithm for creating density-equalizing map projections. *Proceedings of the National Academy of Sciences USA*, **115**:E2156-E2164.
+
+**BibTeX Entry:**
+
+```
+@article{gastner_seguy_more_2018,
+	title={Fast flow-based algorithm for creating density-equalizing map projections},
+	author={Gastner, Michael T. and Seguy, Vivien and More, Pratyush},
+	DOI={10.1073/pnas.1712674115},
+	journal={Proceedings of the National Academy of Sciences of the United States of America},
+	year={2018},
+	volume = {115},
+	number = {10},
+	pages = {E2156--E2164}
+}
+```
+
+## Input data format
+The cartogram generator expects two input files.
+
+1. a '.json' file containing the cartesian coordinates for the map regions under consideration. For example, for the 2016 US presidential election data we provide `usa_contiguous_arcgis.json` (in the `sample_data` folder) which includes the coordinates for the boundaries of the different states of the contiguous United States. You will need to obtain the '.json' file and it needs to adhere to the GeoJSON specifications. It also needs to be processed first by using the command `cartogram -p [json_file_name]` (Instructions found [below](#running-the-cartogram-generator)).  If you do not have a '.json' file, please follow the steps listed here: https://github.com/Flow-Based-Cartograms/go_cart/blob/master/process_map_file.md
+
+2. a '.csv' file containing the data (such as population) for each region, according to which these will be scaled. For the 2016 US presidential election data we provide `usa_contiguous_electors.csv` (in the `sample_data` folder) which provides the number of electors for each state. The csv file will be generated by the cartogram generator when you process the '.json' file. You will then need to add in the data to the csv file.
+
+The `.csv` file should be in the following format:
+```
+Region.Id,Region.Data,Region.Name
+1,9,Alabama
+2,11,Arizona
+3,6,Arkansas
+4,55,California
+5,9,Colorado
+6,7,Connecticut
+7,3,Delaware
+8,3,District of Columbia
+...
+```
+
+## Building the cartogram generator
+
+### Dependencies
+
+#### macOS
+You must have `Xcode Command Line Tools` and the `brew` package manager <a href="https://www.moncefbelyamani.com/how-to-install-xcode-homebrew-git-rvm-ruby-on-mac/" title="How to install Xcode CLT & brew?" target="\_blank">installed</a> (and updated) on your computer.
+
+#### Ubuntu/Debian Linux & Windows 10
+
+No additional dependencies. Your default `apt-get` package manager should work fine.
+
+### Building (Window 10)

```

Comment:
I would put this in the Dependencies section instead of a separate Building section. Also, nit: Windows 10 instead of Window 10.